test/jest/local/Eask

```diff
@@ -27,11 +27,12 @@
 (depends-on "f")
 (depends-on "s")
 (depends-on "fringe-helper")
-(depends-on "watch-cursor"
-            :repo "jcs-elpa/watch-cursor" :fetcher 'github)
-(depends-on "organize-imports-java"
-            :repo "jcs-elpa/organize-imports-java"
-            :fetcher 'github
-            :files '(:defaults "sdk" "default"))
+;; FIXME: including these takes a really long time
```

Comment:
any ideas for more specific tests for recipes like this? excluding improves local run time by like 30%

Reply:
No, unfortunately not. We could use package-vc-install as a replacement, but it requires Emacs 29 or later. 🤔

For now, perhaps just extending the timeout duration would help?

Reply:
> We could use package-vc-install as a replacement

No, I mean we could reduce the number of commands in `local.test.js`, or create a separate test file `recipes.test.js` that just tests commands related to recipe dependencies.

> For now, perhaps just extending the timeout duration would help?

That's the quickest way. It's fine when running as an action, but `local.test.js` often fails locally... it's mostly just annoying.

Reply:
> No, I mean we could reduce the number of commands in local.test.js, or create a separate test file recipes.test.js that just tests commands related to recipe dependencies.

Oh, okay. I think it's fair to have a separate test file. 👍

Reply:
Left it as a FIXME for now, this PR is getting big haha